### ISSUE TITLE
Refdata Pharma File is empty after download

### ISSUE THREAD

**Issue by zdavatz** [zdavatz/cpp2sqlite#27]
```
~/.software/cpp2sqlite> ls -ltr downloads/
insgesamt 1189692
-rw-r--r-- 1 zeno zeno   43030481 27. Feb 13:49 bag_preparations.xml
-rw-r--r-- 1 zeno zeno    2523019 28. Feb 19:20 swisspeddosepublication.xml
-rw-r--r-- 1 zeno zeno 1169794734  1. Mär 01:11 aips.xml
-rw-r--r-- 1 zeno zeno    2877344  1. Mär 11:31 swissmedic_packages.xlsx
-rw-r--r-- 1 zeno zeno          0  1. Mär 11:31 refdata_pharma.xml
```

**Comment by bettar** [comment]
cannot reproduce

**Comment by zdavatz** [comment]
fixed with 9a170e625b325b861ddd03b764e5
